<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_List</class>
 <widget class="QWidget" name="Form_List">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>610</width>
    <height>1086</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>List</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_19">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_15">
       <item>
        <widget class="QLabel" name="label_title">
         <property name="text">
          <string>列表</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_current_item">
         <property name="text">
          <string>当前项</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_check_state">
         <property name="text">
          <string>勾选状态</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_8">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QListWidget" name="listWidget_List">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>2</horstretch>
         <verstretch>2</verstretch>
        </sizepolicy>
       </property>
       <item>
        <property name="text">
         <string>新建项目0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>新建项目4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>a</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>aA</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Aa</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="title">
          <string>输出全部</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButton_print_all">
            <property name="text">
             <string>输出全部</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_print_all">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_16">
         <item>
          <widget class="QGroupBox" name="groupBox_2">
           <property name="title">
            <string>排序</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_17">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_4">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>排序规则</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_ascending_order">
                <property name="text">
                 <string>升序</string>
                </property>
                <property name="checked">
                 <bool>true</bool>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="radioButton_descending_order">
                <property name="text">
                 <string>降序</string>
                </property>
                <attribute name="buttonGroup">
                 <string notr="true">buttonGroup</string>
                </attribute>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_5">
              <item>
               <widget class="QPushButton" name="pushButton_sort">
                <property name="text">
                 <string>排序</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="verticalSpacer_3">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>控制选中项</string>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_18">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <widget class="QPushButton" name="pushButton_copy_current_item">
                <property name="text">
                 <string>复制内容</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_delete_current_row">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_checkable_current_row">
                <property name="text">
                 <string>允许勾选</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_uncheckable_current_row">
                <property name="text">
                 <string>不允许勾选</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_9">
              <item>
               <widget class="QPushButton" name="pushButton_moveToTop">
                <property name="text">
                 <string>移至顶部</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_moveUp">
                <property name="text">
                 <string>上移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_moveDown">
                <property name="text">
                 <string>下移</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_moveToBottom">
                <property name="text">
                 <string>移至底部</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_4">
       <property name="title">
        <string>增</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_7">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="lineEdit_add"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add">
            <property name="text">
             <string>增加</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_add_s">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_add_s">
            <property name="text">
             <string>增加多个</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="lineEdit_insert"/>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_insert_position"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_insert">
            <property name="text">
             <string>插入</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QPlainTextEdit" name="plainTextEdit_insert_s">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
              <horstretch>0</horstretch>
              <verstretch>1</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="spinBox_insert_s_position"/>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_insert_s">
            <property name="text">
             <string>插入多个</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_14">
       <item>
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>内容匹配规则</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_12">
          <item>
           <widget class="QRadioButton" name="radioButton_equal">
            <property name="text">
             <string>精确匹配</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_match</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_include">
            <property name="text">
             <string>包含*</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_match</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_start_with">
            <property name="text">
             <string>以*开头</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_match</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_end_with">
            <property name="text">
             <string>以*结尾</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_match</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_ignore_capltals">
            <property name="text">
             <string>忽略大小写</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_match</string>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButton_regular_expression">
            <property name="text">
             <string>正则匹配</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">buttonGroup_match</string>
            </attribute>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QGroupBox" name="groupBox_5">
           <property name="title">
            <string>删</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_13">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <item>
               <widget class="QLabel" name="label_remove_by_position_result">
                <property name="text">
                 <string>依位置删除 删除结果</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_remove_by_position"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_remove_by_position">
                <property name="text">
                 <string>删除</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <item>
               <widget class="QLabel" name="label_remove_by_content_result">
                <property name="text">
                 <string>依内容删除 删除结果</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_4">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_remove_by_content"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_remove_by_content">
                <property name="text">
                 <string>内容匹配删除</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QPushButton" name="pushButton_clear">
                <property name="text">
                 <string>清空全部</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_7">
           <property name="title">
            <string>改</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_14">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <item>
               <widget class="QLabel" name="label_modify_by_position_result">
                <property name="text">
                 <string>依位置修改 修改结果</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_5">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_modify"/>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_modify_position"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_modify">
                <property name="text">
                 <string>依位置修改</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_replace_by_content_result">
                <property name="text">
                 <string>依内容替换 替换结果</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_6">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_replace_old_content"/>
              </item>
              <item>
               <widget class="QLineEdit" name="lineEdit_replace_new_content"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_replace">
                <property name="text">
                 <string>内容匹配替换</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <item>
               <widget class="QLabel" name="label_change_position_result">
                <property name="text">
                 <string>依位置交换 交换结果</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_change_position_1"/>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_change_position_2"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_change_position">
                <property name="text">
                 <string>依位置交换</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_8">
           <property name="title">
            <string>查</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_15">
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <item>
               <widget class="QLabel" name="label_find_by_position_result">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="text">
                 <string>依位置查找 查找结果</string>
                </property>
               </widget>
              </item>
              <item>
               <spacer name="horizontalSpacer_7">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item>
               <widget class="QSpinBox" name="spinBox_find_result_position">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_find_by_position">
                <property name="text">
                 <string>依位置查找</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <item>
               <widget class="QLineEdit" name="lineEdit_find_by_content"/>
              </item>
              <item>
               <widget class="QLabel" name="label_find_by_content_result">
                <property name="text">
                 <string>依内容查找 查找结果</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_find_by_content">
                <property name="text">
                 <string>依内容查找</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <item>
               <widget class="QLineEdit" name="lineEdit_find_match"/>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton_find_match">
                <property name="text">
                 <string>内容匹配查找</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="verticalLayout_2">
              <item>
               <widget class="QPlainTextEdit" name="plainTextEdit_find_include">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                  <horstretch>0</horstretch>
                  <verstretch>1</verstretch>
                 </sizepolicy>
                </property>
                <property name="plainText">
                 <string>依内容查找 匹配结果</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup_match"/>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
